### ISSUE TITLE
E-mail alerts: change to a Rothamsted email

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#639]
See relevant e-mail chain with ITS (ID:0037906), but the aim is to remove the Farmier e-mail and use a Rothamsted one instead, which Aislinn would maintain/ oversee internally.  There are several benefits to this from both @mstenta/ FarmOS end and getting around the Microsoft Outlook links/ internal visualisation of plain text e-mails.

**Comment by aislinnpearson** [comment]
Slightly delayed by Rothamsted ITS, who are in touch with @mstenta and I. Moving to 2.21 milestone

**Comment by mstenta** [comment]
@aislinnpearson I tried configuring this, but something didn't work. I plugged in all the SMTP credentials and tested by submitting the password reset form at https://northwyke.farmos.net/user/password. The form submitted properly (indicating the the email was able to be sent), but I never received the email.

We may need to talk to IT again and see if they noticed anything come through on their end, and whether or not they can see why it didn't work.
